packages/fxa-content-server/app/scripts/templates/pair/index.mustache

```diff
@@ -1,21 +1,19 @@
 <div id="main-content" class="card">
   <header>
-    <h1 id="pair-header">{{#t}}Connect another device{{/t}}</h1>
+    <h2 id="pair-header">{{#t}}Get Firefox on your phone or tablet{{/t}}</h2>
   </header>
 
   <section>
     <div class="error"></div>
 
-    <div class="graphic {{ graphicId }}" role="img" aria-label="{{#t}}Scan to connect another device{{/t}}"></div>
-    <p>{{#t}}Open the code scanner on your Firefox app and show code when ready.{{/t}}</p>
+    <div class="graphic graphic-connect-another-device-qr-code" role="img" aria-label="{{#t}}Scan to connect another device{{/t}}"></div>
+    <p>{{#t}}Hold your phone's camera over the code to scan.{{/t}}<br>{{#unsafeTranslate}}Or, send yourself a <a href="https://www.mozilla.org/en-US/firefox/mobile/get-app/" class="download-firefox-mobile">download link.{{/unsafeTranslate}}</a></p>
```

Comment:
Whats the purpose of the `#unsafeTranslate` tag?

Reply:
Also, should the closing tag for `{{/unsafeTranslate}}` be left of the closing tag for the anchor?

Reply:
nit: remove the /en-US/ from the URL and let mozilla.org do the locale detection/redirect.

Reply:
Also, not sure if we want to use smart quotes here (`phone’s` above, and `I’m` below on L14).

Reply:
@dschom this page is woefully out of date but we do have some info on `unsafeTranslate` [here](https://mozilla.github.io/ecosystem-platform/reference/localization#a-bit-about-gettext).

Reply:
TIL: We currently have 26 `{{#unsafeTranslate}}` usages in our codebase:
https://gist.github.com/pdehaan/1c6f7e45283f2a5c888cbf51cfc50752

&hellip; and 8 of the variable names don't start w/ "escaped":

> _"Within an `unsafeTranslate tag`, named variables must contain the 'escaped' prefix to remind developers that the variable must be HTML escaped before being written."_
> &mdash; via https://mozilla.github.io/ecosystem-platform/reference/localization#content-server-gotchas

```sh
[packages/fxa-content-server/app/scripts/templates/index.mustache] %(serviceLogo)s is not escaped
[packages/fxa-content-server/app/scripts/templates/index.mustache] %(serviceName)s is not escaped
[packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache] %(email)s is not escaped
[packages/fxa-content-server/app/scripts/templates/pair/supp_allow.mustache] %(email)s is not escaped
[packages/fxa-content-server/app/scripts/templates/ready.mustache] %(serviceLogo)s is not escaped
[packages/fxa-content-server/app/scripts/templates/ready.mustache] %(serviceName)s is not escaped
[packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache] %(serviceLogo)s is not escaped
[packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache] %(serviceName)s is not escape
```

Reply:
Thanks for the links. So I guess maybe it doesn't really matter, but it seems like it would be better if the closing anchor tag was inside the {{#unsafeTranslate}}?

Reply:
Definitely, thank you for seeing that!